components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -303,6 +303,28 @@ Chef Infra Server lets you delete the existing clients one at a time. To delete
 
 {{< figure src="/images/automate/delete-a-client.png" alt="Delete a Client">}}
 
+### Nodes
+
+A [Node]({{< relref "nodes" >}}) is a communication between the Infra Chef Server and the Infra Clients. Nodes are used when an Infra Client asks for a policy from the Infra Chef Server.
+
+The Chef Infra Server UI lets you:
```

Comment:
I know we've discussed this before, I would still love to find a way of phrasing this without `UI`. We should never point out that someone is in a graphical user interface. We should just name it and explain it.

"The Chef Infra Server integration lets you: " or something similar